### ISSUE TITLE
Printing the mode in files

### ISSUE THREAD

**Issue by acbart** [skulpt/skulpt#702]
It seems like writing to files doesn't particularly work outside of stdout, but the repr for functions currently prints the file mode incorrectly. I believe it needs to be converted to JS via `Sk.ffi.remapToJs` first. The relevant line of code: https://github.com/skulpt/skulpt/blob/master/src/file.js#L67

**Comment by albertjan** [comment]
Yeah you're right, both accounts, I ran into that myself yesterday.
